content/en/docs/ops/deployment/vm-architecture/index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Virtual Machine Architecture
+description: Describes Istio's high-level architecture for virtual machines.
+weight: 20
```

Comment:
```suggestion
weight: 25
```

Maybe its 15 so it's placed between Architecture and Deployment Models (already has a 20). Or maybe since you want them to be read first, make it 25. The current placement in the netlify shows it after Deployment models.

Reply:
I still  see them after deployment models in rendered docs.